test.html

```diff
@@ -189,8 +365,17 @@ <h2>Test drive</h2>
 radios[0].onclick();
 
 var textarea = $('textarea', form);
+var savedCode;
+	if (savedCode = window.localStorage ? window.localStorage.getItem('test_code') : null){
+		textarea.textContent = savedCode;
+	}
 
-(textarea.oninput = function() {
+(textarea.onblur = function() {
+	if (window.localStorage){
+		localStorage.setItem('test_code', this.value);
+	}
+},
+textarea.oninput = function() {
```

Comment:
Can we remove the `localStorage` related stuff and split that into a separate PR?

Reply:
We can, but why? :)

Reply:
It's unrelated to the main feature (testing plugins on the test page).